include/swift/AST/DiagnosticsSema.def

```diff
@@ -3516,6 +3516,10 @@ WARNING(pbd_never_used_stmtcond, none,
         "value %0 was defined but never used; consider replacing "
         "with boolean test",
         (Identifier))
+WARNING(unused_implicit_setter_newvalue, none,
+        "implicit setter argument 'newValue' is never used; consider specifying"
+        " a variable name or using the argument `newValue`",
```

Comment:
Per my other commentary, I don't think pinning it to the implicit parameter is quite right. I think it should always apply when the new value parameter is unused but the property is mentioned. Then, the suppression mechanism is the same as for other "unused" warnings:

    _ = newValue

rather than naming something you intentionally won't use.